app/assets/stylesheets/components/availability.scss

```diff
@@ -68,11 +72,41 @@
     }
   }
 
+  .section_heading h3 {
+    // h3 {
+      font-size: 1.1em;
+      margin: 0;
+      line-height: 1.2em;
+    // }
+      background: none;
+      padding-bottom: none;
+  }
+
   .holding-block:last-child, ul li:last-child {
     border: none;
   }
 }
 
+.availability--panel_umlaut {
+  .umlaut-section {
+    border: 2px solid #eee;
+  }
+
+  .section_heading {
+    border-bottom: 2px solid #eee;
+  }
+
+  .umlaut_section_content, .section_heading {
+    padding: 1em;
```

Comment:
Padding for section_heading should be 10px 15px;
Padding for .umlaut_section_content should be 1em;